en/subscription_services_privacy_notice.md

```diff
@@ -1,56 +1,65 @@
+
+__Technical data.__ Mozilla receives basic information about the installed version of your Relay Add-on software, and the device they are installed on, including the operating system and hardware configuration. Across Subscription Services, your IP address is temporarily collected as part of our server logs for 90 days. We use Technical data to improve performance and stability for our users, and to measure the Services’ performance.
+
+__Location data.__ Mozilla Subscription Services receive your IP address when you sign up for and use the service(s). We use the IP address in order to approximate your location because Mozilla Subscription Services availability, pricing and offers may depend on your country. We also use your IP address for fraud and abuse prevention purposes across our products.
+
+__Firefox Relay email messages.__ To send and forward your email messages from your masked email address(es) to your primary email address, Firefox Relay processes your email messages. We do not read or store the content of any of your messages. In the event that an email cannot be delivered to you, we will keep it on our servers and delete it after it has been delivered (in no event will we hold onto it for more than three days). If you use the feature to block promotional emails, the Service will check email headers to determine whether the email should be blocked.
 
-__Masks and where you use them.__ Mozilla maintains a copy of your account information to provide the service, in particular to associate your primary email address with your masked email address(es). If you create a custom mask, Mozilla stores it in order to forward emails to it. Mozilla stores the site where you created the mask, sites where you subsequently use the mask, and any labels associated with the mask in order to ensure that your masks are easily findable when you’re ready to use them. Learn how to [enable and disable these features](https://relay.firefox.com/faq).
+__Firefox Relay Masks (and where you use them).__ Mozilla maintains a copy of your account information to provide the service, in particular to associate your primary email address with your masked email address(es). If you create a custom mask, Mozilla stores it in order to forward emails to it. Mozilla stores the site where you created the mask, sites where you subsequently use the mask, and any labels associated with the mask in order to ensure that your masks are easily findable when you’re ready to use them. Learn how to [enable and disable these features](https://relay.firefox.com/faq).
```

Comment:
lower case `m` in `Masks`?

Reply:
I have made the headings' capitalization consistent. I changed m to lowercase and also the p in phone to lowercase